CMakeLists.txt

```diff
@@ -36,9 +49,11 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 # search for Z3
 find_package(Z3 REQUIRED)
 if (Z3_FOUND AND NOT TARGET z3::z3lib)
-    add_library(z3::z3lib IMPORTED INTERFACE)
+    add_library(z3::z3lib IMPORTED INTERFACE include/Result.h)
```

Comment:
what is this file doing here at the `z3` target?
furthermore, the file isn't even committed.
this should be moved to the `src/CMakeLists.txt`.